tests/ignite/contrib/handlers/test_wandb_logger.py

```diff
@@ -119,7 +119,7 @@ def test_output_handler_metric_names():
     mock_logger.log = MagicMock()
 
     wrapper(mock_engine, mock_logger, Events.ITERATION_STARTED)
-    mock_logger.log.assert_called_once_with({"tag/a": 55.56, "tag/c": "Some text"}, step=7, sync=None)
+    mock_logger.log.assert_called_once_with({"tag/a": 55.56}, step=7, sync=None)
```

Comment:
updated the wandb logger test, updating string value test case. Or, shall we add `isinstance(value, str)` in wandb logger as used in tqdm logger.

Reply:
This is a good question. I was looking at w&b docs if they can accept `str` data on their side. If you can test that explicitly it would be great and if they support str data then it would make sense to add this special case as tqdm...

Reply:
Ok, I will check and make changes accordingly.

Reply:
@vfdev-5 
I found some wandb doc links: https://docs.wandb.ai/guides/track/log, https://docs.wandb.ai/ref/python/log which shows logging of various objects like Image, 3D viz, audio.. etc
However, there does not seem to be any use cases in logging the string data type. 
So I think it would be fine if we exclude the string data type from logging.

Reply:
Yes, I was also inspecting this wandb docs and didn't find on str dtype. We can try to check that explicitly on a testing project to log a str and see what happens. I think they can handle that but metric value as str is something unusual

Reply:
Yes, metric values as str are unusual.

Reply:
I created a hugging face project that trains a Roberta-base on a sequence classification task and tried to log text metrics (eval/text).

```python
def compute_metrics(eval_pred):
        random_str = ["text1", "text2", "text3", "text4", "text5", "text6"]
        pred, act = eval_pred
        return {'score':np.sqrt(mean_squared_error(pred,act)), 'text': random_str[random.randint(0,len(random_str)-1)]}
```

But when I opened the run page (where the plots regarding logged metrics are plotted) I encountered following warning.

String logging.
![Screenshot from 2021-06-02 11-18-48](https://user-images.githubusercontent.com/31338369/120430271-6579f680-c394-11eb-8411-42d3480b481b.png)

Number logging.
![Screenshot from 2021-06-02 11-24-06](https://user-images.githubusercontent.com/31338369/120430914-56e00f00-c395-11eb-8b39-716603ec7d06.png)

However,
we can see the values (eval/text) being logged for string in the overview tab.
![Screenshot from 2021-06-02 11-30-14](https://user-images.githubusercontent.com/31338369/120431364-01f0c880-c396-11eb-86f5-92dbad8050e7.png)

Reply:
Thanks a lot for checking that ! OK, let's add a special case for wandb as for tqdm.